gender_novels/corpus_gen.py

```diff
@@ -303,10 +345,10 @@ def get_publication_date_wikidata(author, title):
             return None
     except (pywikibot.exceptions.NoPage):
         try:
-            if (title == title.split(":", 1)[0]):
+            if (title == title.split(":", 1)[0].split(";,1")[0]):
```

Comment:
I noticed that sometimes the generator would fail to get information for books because the subtitles were included; for example, "A Christmas Carol" is listed as "A Christmas Carol in Prose; Being a Ghost Story of Christmas", so this cuts out the subtitles.  I didn't want to remove them right away though, in case a book actually had colons or semicolons in their titles.